app_config_b2c.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+b2c_tenant = "fabrikamb2c"
+signupsignin_user_flow = "b2c_1_signupsignin1"
+editprofile_user_flow = "b2c_1_profileediting1"
+authority_template = "https://{tenant}.b2clogin.com/{tenant}.onmicrosoftonline.com/{user_flow}"
```

Comment:
>online [](start = 72, length = 6)

`onmicrosoftonline.com`? not just `.onmicrosoft.com`

Reply:
@rayluo  and @abhidnya13 where did you find `onmicrosoftonline.com`? does this sample actually work? you get back a B2C token?

Reply:
My bad. Fixing.

Reply:
lol.

---
In reply to: [341698319](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/4#discussion_r341698319) [](ancestors = 341698319)